test-scripts/cluster-up-minikube.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Copyright (c) 2016-2017 Bitnami
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is taken from https://github.com/bitnami/kubernetes-travis/blob/master/scripts/cluster-up-minikube.sh
+# based on https://engineering.bitnami.com/articles/implementing-kubernetes-integration-tests-in-travis.html#minikube-without-vms.
+
+# From minikube howto
+export MINIKUBE_WANTUPDATENOTIFICATION=false
+export MINIKUBE_WANTREPORTERRORPROMPT=false
+export MINIKUBE_HOME=$HOME
+export CHANGE_MINIKUBE_NONE_USER=true
+mkdir -p ~/.kube
+touch ~/.kube/config
+
+export KUBECONFIG=$HOME/.kube/config
+export PATH=${PATH}:${GOPATH:?}/bin
+
+MINIKUBE_VERSION=v0.22.0
+
+install_bin() {
+    local exe=${1:?}
+    test -n "${TRAVIS}" && sudo install -v ${exe} /usr/local/bin || install ${exe} ${GOPATH:?}/bin
+}
+
+# Travis ubuntu trusty env doesn't have nsenter, needed for VM-less minikube
+# (--vm-driver=none, runs dockerized)
+check_or_build_nsenter() {
+    which nsenter >/dev/null && return 0
+    echo "INFO: Building 'nsenter' ..."
```

Comment:
Do we have to build here. Could we make a container with it already built and just pull it out of there?

Reply:
Seems possible. Is there a convention Travis users for containers vs installs? Not that we need to follow that necessarily. When you say container, is that a travis term for a pre-built image, or are you suggesting a docker image or something similar?

Reply:
Found an image online, so going to go that route.